### PR TITLE
[Merged by Bors] - Consolidate cluster config

### PR DESCRIPTION
**Pull request by maltesander**
# Description

fixes https://github.com/stackabletech/airflow-operator/issues/247

Not quite happy with the `databaseInitialization` field in the `clusterConfig`  (see usage_guide -> logging or logging integration tests). This maybe kept on top level ... but it will be replaced once we use init containers for that anyways.

test: https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/airflow-operator-it-custom/83/

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/airflow/pages/usage.adoc [hunk 1]
```diff
@@ -1,150 +0,0 @@
-= Usage
```
Comment (adwk67) [inline docs/modules/airflow/pages/usage.adoc:1]: This was a duplicate page left over, I guess?

**Review by adwk67 — APPROVED** [review]
Thanks - LGTM! Just one minor question.

File: docs/modules/airflow/pages/usage.adoc (hunk 1 above)
Comment (maltesander) [reply docs/modules/airflow/pages/usage.adoc:1]: Yes

**Comment by adwk67** [comment]
bors merge

**Pull request closed without merging**